src/help/guides/getting-started.md

```diff
@@ -6,99 +6,34 @@ lang: en-US
-1. Download and install [iCloud for Windows 8](https://support.apple.com/en-us/HT204283).
-   <el-tag type="warning">Not the Windows Store app!</el-tag>
-   > Can't find the link to download **iCloud** for Windows 8? <PictureDialog title="Download iCloud for Windows 8" button="GIF" src="/assets/iCloud.gif"/>
-1. Restart your PC after you've installed both **iTunes** and **iCloud**.
-   <el-tag type="warning">You need to restart your computer!</el-tag>
-1. Extract the downloaded **`AltInstaller.zip`** file. Run **`setup.exe`** and follow the instructions.
-1. Launch **AltServer** from your *Notification Area*.
-   > Can't find **AltServer**? <PictureDialog title="Launch AltServer" button="Image" src="/assets/AltServer.png"/>
-1. Connect your **iPhone** or **iPad** to your computer and make sure it's unlocked and your device is trusted.
-1. Open **iTunes** and enable "**iTunes Wi-Fi sync**" for your iPhone/iPad.
-1. Click the **AltServer** icon in your *Notification Area* and click "**Install AltStore**", then select your iPhone/iPad.
-1. Enter your Apple ID email and password. (**BOTH** are case sensitive)
-   > *Your Apple ID and password are sent to Apple, and nowhere else.*
-1. AltStore should be installed to your iPhone/iPad.
-
-### Paperback
-#### Installation
-1. Open **AltServer** along with **iCloud** and **iTunes**.
-1. Make sure you're on the _same connection_ (same router, in most cases) as the computer running **AltServer**. If you're not connected to the same network, you can connect your device via cable.
-1. Trust AltStore application: On your device, go to Settings > General > Device Management. Choose your Apple ID and select **Trust**.
-1. Launch **AltStore** on your iPhone/iPad.
-1. Go to **My Apps** and tap the **+** icon in the top left corner.
-1. Open the **`.IPA`** you downloaded from the [Prerequisites](/help/guides/getting-started/#prerequisites) section.
-1. **Paperback** should finally be installed!
+1. After registering as a [Patreon](https://www.patreon.com/FaizanDurrani) to Paperback, you should receive an email from TestFlight with an invitation to the beta version of Paperback
+   > You may not get your invitation immediately, sometimes it takes a few hours for your registration to be sent to TestFlight. If you have waited over a day, contact us on Discord
+1. Inside of the email, click the button labeled "View in TestFlight"
+1. Download the TestFlight app onto your device from the App Store
+1. At the top of your TestFlight app, there is a "Redeem" button. Select this, and input the code which you have recieved through the email link
+1. Download the application through TestFlight
+   > Whenever there is an update to the beta, you will have to return to TestFlight and select the "Update" button next to the app. It does not do this process automatically. If the setting is enabled, you will get a push notification to your device whenever an update is available
```

Comment:
I believe the notification will come from TestFlight. If I am right shouldn't it be better to explain there is no link between allowing Paperback to send notifications and updates?
```suggestion
   > Whenever there is an update to the beta, you will have to return to TestFlight and select the "Update" button next to the app. It does not do this process automatically. If the setting is enabled for TestFlight, you will get a push notification to your device whenever an update is available
```